lib/puppetserver/ca/host.rb

```diff
@@ -0,0 +1,26 @@
+require 'openssl'
+
+module Puppetserver
+  module Ca
+    class Host
```

Comment:
I guess this is sort of like the client counterpart to the CA object we were discussing. Makes sense.